### ISSUE TITLE
Problem with React 17 and npm version  7.5.1 on mac osx

### ISSUE THREAD

**Issue by pkellner** [buildo/react-placeholder#95]
I've done a clean install of npm 7.5.1 on my MacBook and after trying to do npm install, I get the following message.  I have node 6.14.11 installed on another mac with no issues.  Is there a version problem with the latest node?

```
While resolving: react-components-course@1.0.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from react-placeholder@4.0.3
npm ERR! node_modules/react-placeholder
npm ERR!   react-placeholder@"^4.0.3" from the root project
npm ERR! 
```

**Comment by BrianMitchL** [comment]
The peer dependency of react needs to be updated here, I'll test out react 17 and work on a PR!

**Comment by giogonzo** [comment]
@pkellner @BrianMitchL released as `v4.1.0` on npm
